robocup_ws/src/Planner/src/team_master.py

```diff
@@ -51,12 +68,23 @@ def plan(self):
         if not self.position_updated:
             raise logging.waringn("The simulation state has not beer updated, please run the update_game_state() method before")
             return
+        if not self.players_logic_was_updated:
+            raise logging.waringn("The players logic was not updated - please do so before planning")
+            return
 
         self.actions = [PlayerCommand(0, 0, 0) for _ in range(5)]
 
         self.actions[self.goalkeeper_idx] = self.goalkeeper_logic.get_action(
-            self.team_position.players_positions_efcs[self.goalkeeper_idx], self.team_position.ball_pos_efcs
-        )
+            self.team_position.players_positions_efcs[self.goalkeeper_idx], self.team_position.ball_pos_efcs)
+
+        self.actions[self.striker_left_idx] = self.striker_left_logic.get_action(
+            self.team_position.players_positions_efcs[self.striker_left_idx], self.team_position.ball_pos_efcs)
+        self.actions[self.striker_right_idx] = self.striker_right_logic.get_action(
+            self.team_position.players_positions_efcs[self.striker_right_idx], self.team_position.ball_pos_efcs)
+        self.actions[self.defence_left_idx] = self.defence_left_logic.get_action(
+            self.team_position.players_positions_efcs[self.defence_left_idx], self.team_position.ball_pos_efcs)
+        self.actions[self.defence_right_idx] = self.defence_right_logic.get_action(
+            self.team_position.players_positions_efcs[self.defence_right_idx], self.team_position.ball_pos_efcs)
         self.actions_planned = True
 
     def distribute_goals_to_players(self):   # TODO: how about to rename 'goals -> commends'
```

Comment:
It's fine. commends or commonds? It does make sense to named with distribute (action) commonds to players